corehq/sql_db/operations.py

```diff
@@ -85,7 +87,10 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
             super(RunSqlLazy, self).database_backwards(app_label, schema_editor, from_state, to_state)
 
     def _render_template(self, path):
-        with open(self.sql, encoding='utf-8') as f:
```

Comment:
This erroneously always used `self.sql` (which confusingly is initialized with the forward migration template path, but that's a different issue not addressed in this PR) as the template path rather than the `path` argument provided. The result was that both forward and reverse migrations always used the forward migration template.